docs/DeveloperGuide.md

```diff
@@ -293,24 +293,24 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
-| `* * *`  | new user                | find specific contact(s) by name                                                              | find the person I want to contact                                  |
+| `* * *`  | new user                | find specific contact(s) by name                                                              |                                                                    |
 | `* * *`  | intermediate user       | edit existing contacts                                                                        | update contacts easily without deleting and recreating a new one   |
-| `* * *`  | intermediate user       | filter my contacts by specific tag(s)                                                         | easily find contacts with specific tag(s)                          |
+| `* * *`  | intermediate user       | filter my contacts by specific tag(s)                                                         |                                                                    |
 | `* * *`  | clumsy user             | get a confirmation message when clearing the entire address book                              | avoid accidentally deleting all contacts                           |
 | `* *`    | new user                | see a helpful message when my command is not formatted correctly                              | figure out how to fix my command                                   |
 | `* *`    | new user                | try out the features with some sample data                                                    | understand what each feature does                                  |
 | `* *`    | new user                | tag suppliers with the types of food they supply                                              | easily find the right supplier for specific ingredients            |
 | `* *`    | new user                | tag employees as 'full time' or 'part time'                                                   | easily find the contacts of part time employees during peak period |
-| `* *`    | clumsy user             | see all commands I entered this session                                                       | check what I did previously                                        |
-| `* *`    | clumsy user             | undo my last command                                                                          | undo an erroneous command                                          |
+| `* *`    | clumsy user             | see all commands I entered this session                                                       |                                                                    |
+| `* *`    | clumsy user             | undo my last command                                                                          |                                                                    |
```

Comment:
so that I can rectify an erroneous command

Reply:
Fixed